examples/ansible/setup_gnome_display_manager.yml

```
@@ -0,0 +1,88 @@
+---
+# Ansible Playbook: Deploy and run GNOME Display Manager on ALP Dolomite
+# Description: This Ansible playbook automates the deployment and operation of the GNOME Display Manager (GDM) on SUSE ALP Dolomite using Podman.
+# This deployment allows users to run GDM within a container environment, providing a basic GNOME desktop experience.
+# Based on: "Running the GNOME Display Manager workload using Podman on SUSE ALP Dolomite".
+# Documentation reference: [https://documentation.suse.com/alp/dolomite/html/alp-dolomite/available-alp-workloads.html#task-run-gdm-with-podman]
+
+- name: Deploy and run GNOME Display Manager on ALP Dolomite
+  hosts: alphost
+  become: true
+  vars:
+    workload:
+      name: gdm
+      image: registry.opensuse.org/suse/alp/workloads/tumbleweed_containerfiles/suse/alp/workloads/gdm:latest
+      required_pkgs:
+        - accountsservice
+        - systemd-experimental
+        - python3-selinux
+
+  tasks:
+    - name: Install required packages, if any, for workload {{ workload.name }}
+      ansible.builtin.package:
+        name: "{{ workload.required_pkgs }}"
+        state: present
+      register: required_pkgs_install
+      when:
+        - ((workload.required_pkgs | default([])) | length) > 0
+
+    - name: Handle reboot, if needed
+      when:
+        - required_pkgs_install is changed
+      block:
+
+        - name: Print message requesting immediate reboot
+          ansible.builtin.debug:
+            msg: >-
+              Required system packages have been installed via
+              transactional-update to support workload
+              {{ workload.name }}.
+              An immediate reboot is required to activate these
+              packages before proceding. Once reboot has completed
+              please re-run this playbook.
+
+        - name: Request immediate reboot  # noqa no-handler
+          ansible.builtin.fail:
+            msg: "Reboot required"
+
+    - name: Set SELinux to permissive mode
+      ansible.posix.selinux:
+        policy: targeted
+        state: permissive
+
+    - name: Retrieve image for workload {{ workload.name }}
+      containers.podman.podman_image:
+        name: "{{ workload.image }}"
+        state: present
+
+    - name: Apply container runlabel install for workload {{ workload.name }}
+      ansible.builtin.command: >-
+        podman container runlabel install {{ workload.image }}
+      register: workload_runlabel_install
+      changed_when:
+        - ('already exist' not in workload_runlabel_install.stdout)
+
+    - name: Reload systemd daemon
+      ansible.builtin.systemd:
+        daemon_reload: true
+
+    - name: Reload dbus service
+      ansible.builtin.systemd:
+        name: dbus
+        state: reloaded
+
+    - name: Restart accounts-daemon service
+      ansible.builtin.systemd:
+        name: accounts-daemon
+        state: restarted
```

We just installed it. I suppose it only needs to be running?
```suggestion
        state: started
        enabled: true
```